markets/loggers/loggers.go

```diff
@@ -0,0 +1,29 @@
+package marketevents
+
+import (
+	"github.com/filecoin-project/go-fil-markets/retrievalmarket"
+	"github.com/filecoin-project/go-fil-markets/storagemarket"
+	logging "github.com/ipfs/go-log/v2"
+)
+
+var log = logging.Logger("markets")
+
+// StorageClientLogger logs events from the storage client
+func StorageClientLogger(event storagemarket.ClientEvent, deal storagemarket.ClientDeal) {
+	log.Infow("Storage Event", "Name", storagemarket.ClientEvents[event], "Proposal CID", deal.ProposalCid, "State", storagemarket.DealStates[deal.State], "Message", deal.Message)
```

Comment:
@hannahhoward Could the labels be all lowercased, please?

Reply:
sure will do.